src/celeritas/inp/PhysicsProcess.hh

```diff
@@ -29,25 +29,37 @@ struct BremsProcess
     //! Use integral method for sampling discrete interaction length
     bool integral_xs{true};
 };
+//---------------------------------------------------------------------------//
+/*!
+ * Construct a physics process for electron/positron pair production.
+ */
+struct PairProdProcess
+{
+    //! Bethe-Heitler pair production
+    std::optional<BHPairProdModel> bethe_heitler;
+    //! Muonic pair production
+    std::optional<MuPairProdModel> mu;
+};
 
 //---------------------------------------------------------------------------//
 //!@{
 //! \name Process aliases
 //! \todo rename `em/model` to match, merge muon and electron proceses
 
 using BremsstrahlungProcess = BremsProcess;
+using GammaConversionProcess = PairProdProcess;
+using MuPairProductionProcess = PairProdProcess;
+
 #if 0
 using ComptonProcess = ComptonScatProcess;
 using CoulombScatteringProcess = CoulombScatProcess;
 using EIonizationProcess = IoniProcess;
 using EPlusAnnihilationProcess = AnniProcess;
-using GammaConversionProcess = ConversionProcess;
 using PhotoelectricProcess = PhotoelectricProcess;
 using RayleighProcess = RayleighScatProcess;
 
-using MuBremsstrahlungProcess = MuBremsProcess;
-using MuIonizationProcess = MuIoniProcess;
-using MuPairProductionProcess = MuConversionProcess;
+using MuBremsstrahlungProcess = BremsProcess;
+using MuIonizationProcess = IoniProcess;
```

Comment:
For the processes I sort of prefer the full names (mainly don't love `Anni` or `Scat`):

`ComptonScatteringProcess` or `ComptonProcess`
`CoulombScatteringProcess` or `CoulombProcess`
`RayleighScatteringProcess` or `RayleighProcess`
`AnnihilationProcess`
`BremsstrahlungProcess`
`IonizationProcess`
`PairProductionProcess`
`PhotoelectricProcess`

Reply:
I suspect Geant4 abbreviates `Ioni` because no one was sure whether to use British or American spelling 🙃 And yeah I'm not thrilled about Anni/Scat either but I think the full names are a bit long (and inconsistent if we abbreviate Brems but not the others)

Reply:
Yeah, `WhateverScatteringProcess` probably is too long, and we should be consistent. Though we're not abbreviating photoelectric...

Lengthwise these are pretty similar:

ComptonScatProcess
CoulombScatProcess
RayleighScatProcess
IoniProcess
AnniProcess
BremsProcess
PairProdProcess
PhotoelectricProcess

ComptonProcess
CoulombProcess
RayleighProcess
IonizationProcess
AnnihilationProcess
BremsstrahlungProcess
PairProductionProcess
PhotoelectricProcess

but I think the second is a bit clearer/nicer sounding...

Reply:
If there is only one type of 'SomeName' process, I agree with Amanda that the 2nd list is sufficient.  But for the rare rare (non-existent?) case where there  2 kinds (i.e. both a 'SomeName scattering process' and 'SomeName ionization process' then those process could always a 3rd noun. 

Monkey wrench. What about have fun with namespace?:
```
namespace Process {
   struct Compton { ... };
   struct Photoelectric { ... };
}
```
and hence the name are 
```
Process::Compton
Process::PhotoElectric
```
Probably not as readable but allow (if one wants to) to reduce the number of characters used in some places.

Reply:
One could even have (!?):
```
namespace Process {
namespace Muon {
  struct Bremsstrahlung { ... };
  struct Ionization { ... };
}
}
```

Reply:
Process::Muon vs Muon::Process? 🙃